src/azure-cli/azure/cli/command_modules/backup/custom.py

```diff
@@ -524,10 +526,90 @@ def show_recovery_point(cmd, client, resource_group_name, vault_name, container_
     return client.get(vault_name, resource_group_name, fabric_name, container_uri, item_uri, name)
 
 
+def fetch_tier(paged_recovery_points):
```

Comment:
private function naming should start with underscore. rename it to _fetch_tier

Reply:
done